src/lib/updating-element.ts

```diff
@@ -698,8 +699,8 @@ export abstract class UpdatingElement extends HTMLElement {
    * @param oldValue {any} (optional) old value of requesting property
    * @returns {Promise} A Promise that is resolved when the update completes.
    */
-  requestUpdate(name?: PropertyKey, oldValue?: unknown) {
-    this._requestUpdate(name, oldValue);
+  requestUpdate(name?: PropertyKey, oldValue?: unknown, options?: PropertyDeclaration) {
```

Comment:
This change isn't _strictly_ necessary, right?

Reply:
Do we have cases where element authors would even have a reference to the options?

Reply:
Removed the change to `requestUpdate`.

Reply:
@justinfagnani The use case is overriding `getPropertyDescriptor` (which gets `options`) to return your own setter.  In that case you have `options`, and passing it to `requestUpdateInternal` eliminates the need to do the map lookup.